src/services/module_service.ts

```diff
@@ -94,6 +104,43 @@ export default class ModuleService {
     return importedVersion;
   }
 
+  private static async constructDataForGithubImport(
+    importLine: string,
+  ): Promise<IModule> {
+    const isStd = false;
+    const importUrl: string = importLine.substring(
+      importLine.indexOf("https://"),
+      importLine.indexOf(".ts") + 3, // to include the `.ts`
+    );
+    const importedVersion = ModuleService.getImportedVersionFromImportLine(
+      importLine,
+    );
+    let name = "";
+    const repoNameVersionAndFile =
+      importLine.split("https://raw.githubusercontent.com/")[1];
+    name = repoNameVersionAndFile.split("/")[1];
```

Comment:
Some syntax stuff that could be added just to drop the `let` from this.
```ts
const repoNameVersionAndFile = importLine.split("https://raw.githubusercontent.com/")[1];
const name = repoNameVersionAndFile.split("/")[1] || "";
```

Reply:
Yeah you're completely right actually, i'll make the change just to enforce `const` wherever possible

Reply:
https://github.com/drashland/dmm/pull/91/commits/d24ec69d5dd3e6b44b98376df8330e698e3c8e35